src/storage/sftp/storage.c

```diff
@@ -897,6 +1146,140 @@ storageSftpNew(
                     fingerprint, strZ(param.hostFingerprint));
             }
         }
+        else
+        {
+            LIBSSH2_KNOWNHOSTS *knownHostsList;
+            const char *hostKey;
+            size_t hostKeyLen;
+            int hostKeyType;
```

Comment:
Generally we declare variables as we assign them, rather than all at the top.

Reply:
Updates in
https://github.com/pgbackrest/pgbackrest/pull/2125/commits/4fd017c80716d71a7ffff1f25379b9da196f7001
https://github.com/pgbackrest/pgbackrest/pull/2125/commits/6811405f150c8f1095c3da31d952f0ddebc58be1
https://github.com/pgbackrest/pgbackrest/pull/2125/commits/a3a82046feab0a915e0c116f867818b86ae42621